kdesk/icon-hooks.sh

```diff
@@ -82,7 +82,9 @@ case $icon_name in
     else
         num_file=""
     fi
-    printf "IconStatus: {190,53} $num_file"
+    if [ "$num_file" != "" ]; then
+	printf "IconStatus: {190,53} $num_file\n"
+    fi
```

Comment:
@radujipa I tested this yesterday and couldn't spot any problem, but avoiding it would be cleaner. cc @convolu